.gitignore

```diff
@@ -185,6 +185,9 @@ Session.vim
 *~
 tags
 
+# VS Code
+.vscode
+
```

Comment:
Could you remove this please?

Reply:
I'm using vscode mainly, and don't want to accidentally commit the settings folder

Reply:
yes, that's fine, but you shouldn't push your local .gitignore as not everyone uses .vscode

Reply:
ok, i'll remove it